apps/app-trade-agents/src/utils/constants.ts

```diff
@@ -53,7 +53,7 @@ export const appSettings: Settings = {
   },
   isShowPrevOrderLines: {
     id: 'isShowPrevOrderLines',
-    description: 'Показывать предыдущую заявку',
+    description: 'Показывать предыдущие заявки',
     data: false,
     type: 'boolean',
     sortOrder: 4,
```

Comment:
The code review found some areas for improvement. The 'getStatusColor' function could benefit from better error handling, as the 'default' case currently returns the same status color as 'DRAFT'. Additionally, the comments and descriptions in the 'appSettings' object should be reviewed for accuracy and consistency.

Quality Rate: 6/10